src/LowercaseSentenceTokenizer.java

```diff
@@ -29,8 +30,20 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    * @return a list of tokens, where each token is a word or a period
    */
   public List<String> tokenize(Scanner scanner) {
+    List<String> words = new ArrayList<>();
+    scanner.useDelimiter("\\s+");
```

Comment:
This is actually not needed - Scanner uses whitespace as a delimiter by default.